service/controller/v8patch1/version_bundle.go

```diff
@@ -9,53 +9,17 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "kubernetes",
-				Description: "Update kubernetes to 1.14.3. https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md",
+				Description: "Update kubernetes to 1.14.5 to fix CVE-2019-11247. https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md",
```

Comment:
Why adding this description to the changelog when still the k8scloudconfig is not PRed?